packages/core/helper-plugin/lib/src/components/ReactSelect/ReactSelect.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
```

Comment:
Can you add a story so we know this component is available in the helper plugin's documentation?

Reply:
Done in 06e342814ea6a28a64a075144370a95913ae8eee

Reply:
Well I still need a story inside the helper plugin so we know this component is available.

Reply:
Do you want me to cross-reference the story? Should this really be shown as part of the helper plugin if the component is not actually part of it?